.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: ✅ Setup node
         uses: actions/setup-node@v2
         with:
-          node-version: '17'
+          node-version: '18'
```

Comment:
Dockerfile runs node 17, why did we change it here?

Reply:
We need node 18 to query the predictions endpoint. I put that in the readme. The more detailed explanation: We had issues using the normal http package, so we went with the node-fetch package, but encountered an unfixed import bug, where the author just replied to use fetch, which was implemented with node 18. That's why I changed it here. I changed it in the frontend, so the versions would match

Reply:
I might have missed some places where we set node 17 though, so we would also have to change it there.